apps/backend/src/libs/plugins/authorization-plugin/libs/helpers/check-white-route.helper.ts

```diff
@@ -0,0 +1,9 @@
+const checkWhiteRoute = (route: string, whiteRoutes: string[]): boolean => {
+	return whiteRoutes.some((whiteRoute) => {
+		const regex = new RegExp(`^/api.*${whiteRoute}$`);
```

Comment:
`^\/api\/v\d+.*(\/.+)$`
Then use `route.match(regex)` - the response will be an array, where the second element will be the endpoint (for example /sign-in)
Then check if that endpoint is in white route